backend/src/scripts/utility/scene/removeIncompleteRevisions.js

```diff
@@ -191,6 +192,19 @@ const processTeamspace = async (teamspace, revisionAge) => {
 };
 
 const run = async (revisionAge) => {
+	if (revisionAge < 2) {
+		logger.logWarning(`Revision Age is ${revisionAge}; currently processing jobs will be removed when revision age is set to 0.`);
+
+		const rl = readline.createInterface({ input: process.stdin, output: process.stdout });
```

Comment:
@keifgwinn does reading stdIn works for you? or you rather have a confirmation via extra command argument?

From what I recall you don't log into the machine to run this

Reply:
as discussed, add a flag so skip this if it's in place

Reply:
@carmenfan Added. I've put it inside the `revisionAge < 2` condition so that the warning will still be logged.